### ISSUE TITLE
Ignorar detectar VPN httptoolkit

### ISSUE THREAD

**Issue by brunao-br** [httptoolkit/frida-android-unpinning#90]
@pimterry Muchas aplicaciones detectan la presencia de la VPN. Cuando pruebas estas aplicaciones, ¿usas un script genérico para Frida o siempre creas uno nuevo basado en cada aplicación?

tomando nota de que "poseidontor/android-vpn-detection-bypass" no funcionó.

**Comment by pimterry** [comment]
La verdad es que no he encontrado muchas problemas con detección de VPNs asi personalmente. Imagino que depende del tipo de apps que investigas.

En los pocos casos en que si me causa problemas, suelo interceptar trafico sin el VPN. Simplemente tienes que desactivar el VPN, y hacer la intercepcion con los scripts solos (instrucciones en el README: https://github.com/httptoolkit/frida-interception-and-unpinning?tab=readme-ov-file#android-getting-started-guide). Con los scripts de `android/android-proxy-override.js` y `./native-connect-hook.js` no hace falta el VPN.

Eso funciona para ti?

(Prefieres ingles o español? Veo que lo tradujiste. Me da igual, como prefieras)

**Comment by pimterry** [comment]
Ok. If you use that without the VPN, does it work for you?

I think you might also want `native-connect-hook.js`. This script makes sure that you capture _all_ traffic, even if it tries to ignore the proxy settings.

**Comment by pimterry** [comment]
Yes. All the VPN does is configure proxy settings, and manually rewrite any traffic that tries to ignore those settings. It uses Android's VPN APIs to do this, but it's not a real VPN as such (more info here: https://httptoolkit.com/blog/inspecting-android-http/).

You can use the scripts here to set proxy settings (android-proxy-override.js) & manually redirect traffic (native-connect-hook.js) and then the VPN app isn't required.
